dipy/workflows/reconst.py

```diff
@@ -815,6 +818,9 @@ def run(self, input_files, bvalues_files, bvectors_files, mask_files,
            estimation of tensors by outlier rejection. MRM, 53: 1088-95.
 
         """
+        if save_metrics is None:
+            save_metrics = []
```

Comment:
`save_metrics = save_metrics or []` instead

Reply:
Done in f49517e.